<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SenderWindow</class>
 <widget class="QMainWindow" name="SenderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paging Transmitter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="msgTxt">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="senderCmb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="receiverCmb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="recordBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Playback</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendAudioBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="emergBrdLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Emergency Broadcast</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendMsgBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Message</string>
    </property>
   </widget>
   <widget class="QLabel" name="senderIdLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sender ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="receiverIdLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Receiver ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addSenderBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Sender</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addReceiverBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Receiver</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sampleCmb">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>4000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16000</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="brdSampleLbl">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sampling Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="baudRateLbl">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud Rate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="baudRateCmb">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="msgPriorityLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Message Priority</string>
    </property>
   </widget>
   <widget class="QComboBox" name="msgPriorityCmb">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="testMenu">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Testing Menu</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="headerChk">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Headers</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="poissonBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Poisson</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
